zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -112,14 +109,7 @@
 (
     TuyaQuirkBuilder("_TZE200_pay2byax", "TS0601")  # Cusam ZG-102ZL
     .applies_to("_TZE200_n8dljorx", "TS0601")
-    .tuya_sensor(
-        dp_id=101,
-        attribute_name="measured_value",
-        type=t.uint16_t,
-        fallback_name="Illuminance",
-        device_class=SensorDeviceClass.ILLUMINANCE,
-        state_class=SensorStateClass.MEASUREMENT,
-    )
+    .tuya_illuminance(dp_id=101)
```

Comment:
There are other references in `ts0601_motion` that we could update on the same PR.